test/evp_libctx_test.c

```diff
@@ -677,6 +677,7 @@ static int kem_rsa_params(void)
           && TEST_int_eq(EVP_PKEY_CTX_set_kem_op(NULL,  NULL), 0)
           /* Test secretlen is optional */
           && TEST_int_eq(EVP_PKEY_CTX_set_kem_op(pubctx, "RSASVE"), 1)
+          && TEST_int_eq(EVP_PKEY_encapsulate(pubctx, NULL, &ctlen, NULL, NULL), 1)
```

Comment:
How is this test change hitting the failure cases?

Reply:
Its not.  This line was added because on line 681 we call:
```
TEST_int_eq(EVP_PKEY_encapsulate(pubctx, ct, &ctlen, secret, NULL), 1)
```
With this change, because ctlen is not set to the minimum buffer size required, the call on line 681 will fail.  The call on 680 is just used to fetch the minimum value of wrappedlen.  We already dead recon it of course as we know what RSA_size is, this is just an easy way to fetch it within this cascading if conditional

Reply:
The code you added to raise errors on line 296 and 376 of rsa_kem.c should have coverage tests.